From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 4 Feb 2018 18:51:13 +0100
Subject: [PATCH] {MAYBE UPS} fs/xfs: silence clang's -Wmissing-braces

clang complains:

    fs/xfs/scrub/agheader.c:770:37: error: suggest braces around
    initialization of subobject [-Werror,-Wmissing-braces]
            struct xfs_scrub_agfl_info      sai = { 0 };
                                                    ^
                                                    {}
---
 fs/xfs/scrub/agheader.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/xfs/scrub/agheader.c b/fs/xfs/scrub/agheader.c
index fd975524f460..9bcfdb0d1da3 100644
--- a/fs/xfs/scrub/agheader.c
+++ b/fs/xfs/scrub/agheader.c
@@ -767,7 +767,7 @@ int
 xfs_scrub_agfl(
 	struct xfs_scrub_context	*sc)
 {
-	struct xfs_scrub_agfl_info	sai = { 0 };
+	struct xfs_scrub_agfl_info	sai = { { 0 } };
 	struct xfs_agf			*agf;
 	xfs_agnumber_t			agno;
 	unsigned int			agflcount;
-- 
